### PR TITLE
NIFI-10628: Added Default Schedule Time and Run Duration to processors

### PR DESCRIPTION
**Pull request by Lehel44**
# Summary

[NIFI-10628](https://issues.apache.org/jira/browse/NIFI-10628)

# Tracking

Please complete the following tracking steps prior to pull request creation.

### Issue Tracking

- [x] [Apache NiFi Jira](https://issues.apache.org/jira/browse/NIFI-10628) issue created

### Pull Request Tracking

- [x] Pull Request title starts with Apache NiFi Jira issue number, such as `NIFI-00000`
- [x] Pull Request commit message starts with Apache NiFi Jira issue number, as such `NIFI-00000`

### Pull Request Formatting

- [x] Pull Request based on current revision of the `main` branch
- [x] Pull Request refers to a feature branch with one commit containing changes

# Verification

Please indicate the verification steps performed prior to pull request creation.

### Build

- [ ] Build completed using `mvn clean install -P contrib-check`
  - [ ] JDK 8
  - [ ] JDK 11
  - [ ] JDK 17

### Licensing

- [ ] New dependencies are compatible with the [Apache License 2.0](https://apache.org/licenses/LICENSE-2.0) according to the [License Policy](https://www.apache.org/legal/resolved.html)
- [ ] New dependencies are documented in applicable `LICENSE` and `NOTICE` files

### Documentation

- [ ] Documentation formatting appears as expected in rendered files

### CODE REVIEW

File: nifi-nar-bundles/nifi-aws-bundle/nifi-aws-processors/src/main/java/org/apache/nifi/processors/aws/sqs/GetSQS.java [hunk 1]
```diff
@@ -64,6 +66,7 @@
     @WritesAttribute(attribute = "sqs.message.id", description = "The unique identifier of the SQS message"),
     @WritesAttribute(attribute = "sqs.receipt.handle", description = "The SQS Receipt Handle that is to be used to delete the message from the queue")
 })
+@DefaultSchedule(strategy = SchedulingStrategy.TIMER_DRIVEN, period = "1 min")
```
Comment (exceptionfactory) [inline nifi-nar-bundles/nifi-aws-bundle/nifi-aws-processors/src/main/java/org/apache/nifi/processors/aws/sqs/GetSQS.java:69]: Similar to `GetAzureEventHub`, it seems like this Processor needs to be run more often. Recommend removing the annotation from this set of changes and revisiting.

**Review by exceptionfactory — CHANGES_REQUESTED** [review]
Thanks for introducing these new default settings @Lehel44. The majority of the settings look good, but there are several event-handling Processors where the 1 minute setting does not seem optimal. In those cases, it seems better to avoid the annotation for now.

**Comment by turcsanyip** [comment]
@Lehel44 I checked the processors in the PR and `GetTwitter` and `ConsumeWindowsEventLog` are also consumer-type processors (similar to `GetAzureEventHub`) where an asynchronous thread feeds an internal queue and `onTrigger()` should consume it quite frequently. So I do not recommend 1 min schedule definitely. Please remove these processors from this round. We can consider to optimize the scheduling later.

**Review by exceptionfactory — COMMENTED** [review]
Thanks for making the updates @Lehel44, I agree with @turcsanyip that the annotations should be removed from `GetTwitter` and `ConsumeWindowsEventLog` for now. With those changes, I am ready to approve and merge the PR.

**Comment by Lehel44** [comment]
Thanks @turcsanyip @exceptionfactory for the review. I updated the mentioned processors and removed the annotations.

**Review by exceptionfactory — APPROVED** [review]
Thanks for the latest updates @Lehel44! +1 merging

**Pull request closed without merging**